ui/src/app/applications/components/applications-list/applications-table.tsx

```diff
@@ -71,6 +72,8 @@ export const ApplicationsTable = (props: {
                                 <AppUtils.HealthStatusIcon state={app.status.health} /> <span>{app.status.health.status}</span>
                                 <br />
                                 <AppUtils.ComparisonStatusIcon status={app.status.sync.status} /> <span>{app.status.sync.status}</span>
+                                <br />
```

Comment:
I think we should avoid adding the third row in the table row. It will add unused white space and unsynced/synced app rows are going to have different height. Instead, I would show operation status next to sync status icon in the same line.